test/integration/web_console_access_test.go

```diff
@@ -17,7 +22,22 @@ import (
 	testserver "github.com/openshift/origin/test/util/server"
 )
 
-func tryAccessURL(t *testing.T, url string, expectedStatus int, expectedRedirectLocation string) *http.Response {
+func templateEscape(s string) (string, error) {
+	temp := new(bytes.Buffer)
+
+	t, err := template.New("foo").Parse(`{{define "T"}}{{.}}{{end}}`)
```

Comment:
this doesn't do what you think it does... html/template is context aware. to get the same escaping as in the login page template, you need `{{define "T"}}<a href="{{.}}"></a>{{end}}`